module/data/actor/templates/details.mjs

```diff
@@ -0,0 +1,37 @@
+/**
+ * Shared contents of the details schema between various actor types.
+ */
+export default class DetailsField {
```

Comment:
It feels a little awkward using a class in this way. It feels like we're not really saving that much in terms of lines of code compared to the case where we just declare these explicitly inline. Could we move to do that, even if we repeat some of this for character/npc/etc?

Reply:
Not only LoC though, it's about code reuse as well, and not having to make the same change in 3/4 separate places.